src/freeclimb.Test/Enums/AccountStatusTests.cs

```diff
@@ -0,0 +1,166 @@
+/*
+ * FreeClimb API
+ *
+ * FreeClimb is a cloud-based application programming interface (API) that puts the power of the Vail platform in your hands. FreeClimb simplifies the process of creating applications that can use a full range of telephony features without requiring specialized or on-site telephony equipment. Using the FreeClimb REST API to write applications is easy! You have the option to use the language of your choice or hit the API directly. Your application can execute a command by issuing a RESTful request to the FreeClimb API. The base URL to send HTTP requests to the FreeClimb REST API is: /apiserver. FreeClimb authenticates and processes your request.
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * Contact: support@freeclimb.com
+ * Generated by: https://github.com/openapitools/openapi-generator.git
+ */
+
+
+
+
+using Xunit;
+
+using System;
+using System.Linq;
+using System.IO;
+using System.Collections.Generic;
+using freeclimb.Api;
+using freeclimb.Model;
+using freeclimb.Client;
+using System.Reflection;
+using Newtonsoft.Json;
+
+using freeclimb.Enums;
+
+namespace freeclimb.Test.Enums
+{
+    /// <summary>
+    ///  Class for testing AccountStatus
+    /// </summary>
+    /// <remarks>
+    /// This file is automatically generated by OpenAPI Generator (https://openapi-generator.tech).
+    /// Please update the test case below to test the model.
+    /// </remarks>
+    public class AccountStatusTests : IDisposable
+    {
+        // TODO uncomment below to declare an instance variable for AccountStatus
+        private AccountStatus instance;
+
+        public AccountStatusTests()
+        {
+            instance = AccountStatusInstanceValue();
+        }
+
+        public void Dispose()
+        {
+            // Cleanup when everything is done.
+        }
+
+        /// <summary>
+        /// Test an instance of AccountStatus
+        /// </summary>
+        [Fact]
+        public void AccountStatusInstanceTest()
+        {
+            // TODO uncomment below to test "IsType" AccountStatus
+            //Assert.IsType<AccountStatus>(instance);
+        }
+
+
+        [Fact]
+        public void EnumToStringValueTest()
+        {
+            string expectedValue = StringValueAccountStatusTestValue();
+            string calculatedValue = Extensions.GetMemberValue(instance);
+            Assert.Equal(expectedValue, calculatedValue);
+        }
+        [Fact]
+        public void StringToEnumValueTest()
+        {
+            string expectedValue = StringValueAccountStatusTestValue();
+            AccountStatus calculatedValue = Reflection.GetEnumByValue<AccountStatus>(expectedValue);
+            Assert.Equal(instance, calculatedValue);
+        }
```

Comment:
rather than having a generitc string -> enum, enum -> string test, can we be explicit and have tests for converting each property thats on the enum?